src/pkg/packager/lint/lint_test.go

```diff
@@ -110,4 +149,48 @@ func TestValidateSchema(t *testing.T) {
 		acutal := makeFieldPathYqCompat(input)
 		require.Equal(t, input, acutal)
 	})
+
+	t.Run("isImagePinned", func(t *testing.T) {
+		t.Parallel()
+		tests := []struct {
+			input    string
+			expected bool
+			err      error
+		}{
+			{
+				input:    "registry.com:8080/defenseunicorns/whatever",
+				expected: false,
+				err:      nil,
+			},
+			{
+				input:    "ghcr.io/defenseunicorns/pepr/controller:v0.15.0",
+				expected: false,
+				err:      nil,
+			},
+			{
+				input:    "busybox:latest@sha256:3fbc632167424a6d997e74f52b878d7cc478225cffac6bc977eedfe51c7f4e79",
```

Comment:
We should add this test case to `ParseImageRef`'s test as well

Reply:
Ahhhh now I get what you mean, I hadn't realized that was a function that we were maintaining